components/select-chain-dropdown.tsx

```diff
@@ -89,19 +92,26 @@ export default function SelectChainDropdown({
     setSelectedChain(chainToOption(chain));
   }
 
-  function updateOptions() {
+  async function updateOptions() {
     if (!supportedChains || (isOnNetwork && !Service?.network?.availableChains)) return;
 
     const configuredChains = supportedChains.filter(isChainConfigured);
```

Comment:
this variable became redundant, and the filter can be made _before_ the map on the (new) lime 100